src/callback.h

```diff
@@ -72,7 +72,7 @@ class WesolowskiCallback :public INUDUPLListener {
 
 class OneWesolowskiCallback: public WesolowskiCallback {
   public:
-    OneWesolowskiCallback(integer& D, uint64_t wanted_iter) : WesolowskiCallback(D) {
+    OneWesolowskiCallback(integer& D, form& f, uint64_t wanted_iter) : WesolowskiCallback(D) {
```

Comment:
is this an in-out parameter?
I would expect it to be a `const&` if it's a pure in-parameter, or by value `Form` if it's movable